src/django-picklefield/constants.py

```diff
@@ -0,0 +1 @@
+DEFAULT_PROTOCOL = 2
```

Comment:
the package name needs to be an underscore -- that's why the testsuite is failing